extensions/net/lib.rs

```diff
@@ -89,7 +89,16 @@ pub fn get_unstable_declaration() -> PathBuf {
   PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("lib.deno_net.unstable.d.ts")
 }
 
-pub fn init<P: NetPermissions + 'static>(unstable: bool) -> Extension {
+/// `NoCertificateValidation` is a wrapper struct so it can be placed inside `GothamState`;
+/// using type alias for a `Option<Vec<String>>` could work, but there's a high chance
+/// that there might be another type alias pointing to a `Option<Vec<String>>`, which
+/// would override previously used alias.
+pub struct NoCertificateValidation(Option<Vec<String>>);
```

Comment:
We have duplicate at weboscket now, should be that handled somehow so that we have only one struct definition?

Reply:
No, this is correct, if you need to place multiple things in `GothamState` that are of the same type, then using a wrapper struct like here is the correct solution.